DataProviderService/StarSystemSqLiteRepository.cs

```diff
@@ -272,6 +273,7 @@ public List<StarSystem> GetStarSystems(string[] names, bool refreshIfOutdated =
             if (systemsToUpdate.Count > 0)
             {
                 List<StarSystem> updatedSystems = dataProviderService.GetSystemsData(systemsToUpdate.Select(s => s.Key).ToArray());
+                if (updatedSystems == null) { return null; }
```

Comment:
Deserialize and return the old results rather than null?

Reply:
I was aiming for minimal changes.